users/serializers.py

```diff
@@ -100,6 +104,7 @@ def validate(self, data: Dict[str, Any]) -> Dict[str, Any]:  # pylint: disable=n
         return data
 
     def get_cleaned_data(self) -> Dict[str, str]:
+        logger.info("Get cleaned data method in CustomRegisterSerializer")
```

Comment:
I don't think this is needed as `info` here. Maybe change it to debug? What do you think @rwrzesien ?

Reply:
I've changed it to debug.